public/app/features/explore/QueryRow.tsx

```diff
@@ -38,6 +38,12 @@ export class QueryRow extends PureComponent<QueryRowProps, QueryRowState> {
     this.props.runQueries(exploreId);
   };
 
+  onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Enter' && (event.shiftKey || event.ctrlKey)) {
```

Comment:
Does cmd+enter run queries anywhere else in the codebase? If we do, `event.metaKey` could be added to the shift/ctrl OR to allow for that.

Reply:
From what i've seen cmd+enter doesn't run any queries in the codebase but i guess it wouldn't hurt to add it.

Reply:
Tested Prometheus and cmd+enter adds a newline, so many it shouldn't be added if nowhere else does. I did search for `event.metaKey` and it's used several times in the codebase but I guess none of those were query editors.

Reply:
Yeah you're right. Then i think we should exclude it and stick with <kbd>Ctrl</kbd> and <kbd>Shift</kbd>